code/game/gamemodes/blob/overmind.dm

```diff
@@ -90,7 +91,7 @@
 		return
 
 	var/message_a = say_quote(message, get_spans())
-	var/rendered = "<font color=\"#EE4000\"><i><span class='game say'>Blob Telepathy, <span class='name'>[name]([blob_reagent_datum.name])</span> <span class='message'>[message_a]</span></span></i></font>"

```

Comment:
Fun fact; 'game say' and 'message' don't appear to be actual span classes.
'name' is just a fancy way to say bold, too.